### PR TITLE
TOOLS-2543 MNX Tooling updates

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

**Review by danmcd — DISMISSED** [review]
Ship it, and don't forget to merge into release-20220505! :)

**Review by danmcd — COMMENTED** [review]
We don't need to bump this micro-release now, do we?

**Review by danmcd — APPROVED** [review]
Brian told me offline:  No.

Ship it!

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "registrar",
     "description": "Agent to register with binder",
     "version": "1.0.1",
```
[the remark this answers is not in the document]
Comment (bahamat) [reply package.json:4]: Correct.

**Pull request merged**